.buildkite/blueoil-pipeline.yml

```diff
@@ -12,33 +12,123 @@ steps:
+    artifact_paths:
+    - "tmp/*/export/*/*/output/**/*.py"
+    - "tmp/*/export/*/*/output/**/*.yaml"
+    - "tmp/*/export/*/*/output/**/*.so"
+    - "tmp/*/export/*/*/inference_test_data/raw_image.png"
   - command: "make test-semantic-segmentation"
     label: "semantic-segmentation"
+    key: "semantic-segmentation"
     agents:
     - "agent-type=gpu"
     - "env=production"
     timeout_in_minutes: "30"
     env:
       BUILDKITE_CLEAN_CHECKOUT: 'true'
+    artifact_paths:
+    - "tmp/*/export/*/*/output/**/*.py"
+    - "tmp/*/export/*/*/output/**/*.yaml"
+    - "tmp/*/export/*/*/output/**/*.so"
+    - "tmp/*/export/*/*/inference_test_data/raw_image.png"
   - command: "make test-keypoint-detection"
-    label: "keypoint_detection"
+    label: "keypoint-detection"
+    key: "keypoint-detection"
     agents:
     - "agent-type=gpu"
     - "env=production"
     timeout_in_minutes: "30"
     env:
       BUILDKITE_CLEAN_CHECKOUT: 'true'
+    artifact_paths:
+    - "tmp/*/export/*/*/output/**/*.py"
+    - "tmp/*/export/*/*/output/**/*.yaml"
+    - "tmp/*/export/*/*/output/**/*.so"
+    - "tmp/*/export/*/*/inference_test_data/raw_image.png"
+  - command: |
+      buildkite-agent artifact download "tmp/*" ./ --build \${BUILDKITE_BUILD_ID}
+      export CWD=\$(pwd)
+      for TEST_CASE in \$(ls tmp/)
+      do
+        echo \${TEST_CASE}
+        cd tmp/\${TEST_CASE}/export/*/*/output/python
+        python3 run.py -i ../../inference_test_data/raw_image.png -c ../models/meta.yaml -m ../models/lib/libdlk_x86.so
+        cd \${CWD}
+      done
+    label: "inference on ubuntu(x86)"
+    depends_on:
+    - "classification"
+    - "object-detection"
+    - "semantic-segmentation"
+    - "keypoint-detection"
+    agents:
+    - "agent-type=normal"
+    - "env=production"
+    timeout_in_minutes: "30"
+    env:
+      BUILDKITE_CLEAN_CHECKOUT: 'true'
+  - command: |
+      buildkite-agent artifact download "tmp/*" ./ --build \${BUILDKITE_BUILD_ID}
+      export CWD=\$(pwd)
+      for TEST_CASE in \$(ls tmp/)
+      do
+        echo \${TEST_CASE}
+        cd tmp/\${TEST_CASE}/export/*/*/output/python
+        python3 run.py -i ../../inference_test_data/raw_image.png -c ../models/meta.yaml -m ../models/lib/libdlk_aarch64.so
+        cd \${CWD}
+      done
+    label: "inference on raspberry-pi(aarch64)"
+    depends_on:
+    - "classification"
+    - "object-detection"
+    - "semantic-segmentation"
+    - "keypoint-detection"
+    agents:
+    - "agent-type=raspberry-pi"
+    - "env=benchmark"
+    timeout_in_minutes: "30"
+    env:
+      BUILDKITE_CLEAN_CHECKOUT: 'true'
+  - command: |
+      buildkite-agent artifact download "tmp/*" ./ --build \${BUILDKITE_BUILD_ID}
+      export CWD=\$(pwd)
+      for TEST_CASE in \$(ls tmp/)
+      do
+        echo \${TEST_CASE}
+        cd tmp/\${TEST_CASE}/export/*/*/output/python
+        sudo python run.py -i ../../inference_test_data/raw_image.png -c ../models/meta.yaml -m ../models/lib/libdlk_arm.so
```

Comment:
Do we need `sudo` to run with `libdlk_arm.so`? I thought it's needed only for `libdlk_fpga.so`.